### DISCUSSION TITLE
"No native ExponentConstants module found" in loading a tensorflow.js model with  '@tensorflow/tfjs'

### DISCUSSION THREAD

**Discussion by takunda2013** [react-native-community/cli#2398] [Q&A] [answered]
**I am getting these warnings when l try to load a tensorflow.js model using** "await tf.loadLayersModel(
     bundleResourceIO(modelJson, modelWeights),"

 _LOG  Running "AwesomeProject" with {"rootTag":11}
 WARN  The "EXNativeModulesProxy" native module is not exported through NativeModules; verify that expo-modules-core's native code is linked properly
 WARN  No native ExponentConstants module found, are you sure the expo-constants's module is linked properly?_

**Comment by szymonrybczak** [comment, marked as answer]
hey @takunda2013, it looks like you have troubles with Expo Modules. This repository is not the right place for this kind of issues.
